extension/test/integration/make-payment.handler.spec.js

```diff
@@ -95,7 +100,7 @@ describe('::make-payment with multiple adyen accounts use case::', () => {
         interaction.fields.type === constants.CTP_INTERACTION_TYPE_MAKE_PAYMENT
     )
     const makePaymentRequest = JSON.parse(interfaceInteraction.fields.request)
-    expect(makePaymentRequest.metadata).to.deep.equal({
+    expect(makePaymentRequest.metadata).to.deep.include({
```

Comment:
Can you check that it contains also other metadata?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/883/commits/001697eb49d7d950cbe254e37b1aab7ff4fc5ec7